addons/aushelper/bootstrap.js

```diff
@@ -55,7 +57,7 @@ function startup() {
     wrk = Cc["@mozilla.org/windows-registry-key;1"].createInstance(Ci.nsIWindowsRegKey);
     wrk.open(wrk.ROOT_KEY_LOCAL_MACHINE, regCPUPath, wrk.ACCESS_READ);
   } catch (e) {
-    Cu.reportError("AUSHelper - unable to open registry. Exception: " + e);
+    Cu.reportError(`AUSHelper - unable to open registry. Exception: ${e}`);
```

Comment:
Why these changes? Is this a requirement due to an eslint rule? If yes I don't think this particular rule should be enabled :)